### PR TITLE
Lambda telemetry extension for aws

### PR DESCRIPTION
**Pull request by bradleycamacho**
Adds a new doc written by @4demos

### CODE REVIEW

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Amazon Lambda Telemetry integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's Amazon App Runner monitoring integration: what data it reports, and how to enable it."
```
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:7]: why App Runner? Lambda telemetry extension is another project, nothing common with App Runner

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+ # New Relic AWS Lambda Telemetry extension
+
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch.Send your logs, events, metrics and traces by simply adding the extension as the layer and inputting your New Relic license key.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:12]:
```suggestion
The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by simply adding the extension as the layer and inputting your New Relic license key.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Installation
+
+1. In the AWS Console go to **Lambda** > **Functions** then choose the function you'd like to ingest data from.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:16]:
```suggestion
1. In the AWS Console go to **Lambda** > **Functions** and choose the function you'd like to ingest data from.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+2. Select **Layers** > **Add Layer**.
+3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
+4. Put `arn:aws:lambda:us-west-2:533243300146:layer:python-telemetry-api-extension:<version>` in the field, then click **Add**.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:19]:
```suggestion
4. Copy and paste `arn:aws:lambda:us-west-2:533243300146:layer:python-telemetry-api-extension:<version>` in the field, then click **Add**.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+5. Go to **configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:20]: If it's capitalized in the UI...
```suggestion
5. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's Amazon App Runner monitoring integration: what data it reports, and how to enable it."
+---
+
+ # New Relic AWS Lambda Telemetry extension
+
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch.Send your logs, events, metrics and traces by simply adding the extension as the layer and inputting your New Relic license key.
+
```
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:13]: add space in "Amazon CloudWatch.Send"

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+'initDurationMs'
+'memorySizeMB'
+'maxMemoryUsedMB'
+'producedBytes'
+```
+
+### Events
+
+To see event sent by this extension, in **Custom events** section look for `Lambda_Ext_` prefix. Currently there are
+twelve event types:
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:54]: Maybe backspace before "twelve" so this isn't on its own line.

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+platform_extension
+platform_telemetrySubscription
+platform_logsDropped
+function
+extension
+```
+
+### Traces
+
+Traces data are displayed in **Traces** tab. Search for `python-telemetry-api-extension' to filter for these traces.
```
Comment (a-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:73]:
```suggestion
Trace data are displayed in **Traces** tab. Search for `python-telemetry-api-extension' to filter for these traces.
```

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 4 above)
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx]: the name of the function has changed, now it is  `arn:aws:lambda:us-west-2:533243300146:layer:AwsLambdaExtension:<version>`

**Review by a-sassman — COMMENTED** [review]
Looks good! Just a few minor copy edits and suggestions for rewording.

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch.Send your logs, events, metrics and traces by simply adding the extension as the layer and inputting your New Relic license key.
+
+## Installation
+
+1. In the AWS Console go to **Lambda** > **Functions** then choose the function you'd like to ingest data from.
+2. Select **Layers** > **Add Layer**.
+3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
+4. Put `arn:aws:lambda:us-west-2:533243300146:layer:python-telemetry-api-extension:<version>` in the field, then click **Add**.
+5. Go to **configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+
```
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:21]: we need to mention that currently set up has to be done in us-west-2 (Oregon) region

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+After the test has completed, you can find your data on New Relic by navigating to **[https://one.newrelic.com/](https://one.newrelic.com/)** > **Browse data** > then selecting one of 4 categories: **Logs, Events, Metrics or Traces**.
+
+### Logs:
+
+Logs can be filtered by specific attributes set. Click **Attributes** in left menu and then choose the attributes of interest. For example:
+
+```
+SOURCE > api.logs
+PLUGIN > TYPE > lambda extension
+AWS > EXTENSION > python-telemetry-api-extension
```
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:35]: AwsLambdaExtension should replace python-telemetry-api-extension

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx (hunk 8 above)
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx:73]: AwsLambdaExtension should replace python-telemetry-api-extension
I's say "to filter for the relevant traces."

**Review by 4demos — CHANGES_REQUESTED** [review]

**Review by 4demos — DISMISSED** [review]

**Review by a-sassman — APPROVED** [review]

**Pull request merged**